### PR TITLE
[DO NOT MERGE] README: warning about obsoleted master branch

### PR DESCRIPTION
**Pull request by kolyshkin**
#### What does this pull request do?

This is not a pull request, but rather a way to discuss the wording in README.md for the
proposed `master` branch, needed to alleviate #426.

Once the wording is agreed upon, I will create a new `master` branch in this repo.

#### Where should the reviewer start?

#426

#### How should this be manually tested?

n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,130 +1,20 @@
...
-[![GoDoc](https://godoc.org/github.com/fsnotify/fsnotify?status.svg)](https://godoc.org/github.com/fsnotify/fsnotify) [![Go Report Card](https://goreportcard.com/badge/github.com/fsnotify/fsnotify)](https://goreportcard.com/report/github.com/fsnotify/fsnotify)
+If you are reading this, you use `master` branch of this repository,
+which is wrong.
 
-fsnotify utilizes [golang.org/x/sys](https://godoc.org/golang.org/x/sys) rather than `syscall` from the standard library. Ensure you have the latest version installed by running:
+This branch
+ - should not be used;
+ - is not maintained;
+ - is not supported;
+ - will be removed soon.
```
Comment (nathany) [inline README.md:10]: Thanks @kolyshkin 

Do we want to say "soon"? I'm not sure when it will be removed...

**Review by nathany — COMMENTED** [review]
Thanks for doing this. It looks reasonable, and will hopefully alleviate issues for some people until they update things.

Wondering if we should include the "how to update your local branch" text in the readme on master? Or do you think it's unnecessary (given their use case)?
https://github.com/fsnotify/fsnotify/issues/426#issuecomment-1034067268

File: README.md (hunk 1 above)
Comment (kolyshkin) [reply README.md:10]: I wrote "in 2024" initially, but when I looked at it from user's perspective, I realized that "in 2024" reads like "there's nothing to worry about, you can relax for now" which is not the message I want to send.

So, I'd rather say "soon", having in mind a very broad definition of the term. :)

I can remove it if you feel it doesn't work.

Comment (nathany) [reply README.md:10]: That makes sense. Let's keep "soon"

**Comment by kolyshkin** [comment]
> Wondering if we should include the "how to update your local branch" text in the readme on master? Or do you think it's unnecessary (given their use case)?

Makes sense, thanks, added!

**Pull request closed without merging**